src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -602,7 +610,7 @@ module.exports = class LevelPlane {
     for (let i = blocksPositions.length - 1; i >= 0; --i) {
       let destination = [blocksPositions[i][0] + offsetX, blocksPositions[i][1] + offsetY];
       let block = this.getBlockAt(blocksPositions[i]);
-      if (this.inBounds(destination) && this.getBlockAt(destination).isDestroyableUponPush()) {
+      if ((this.inBounds(destination) && this.getBlockAt(destination).isDestroyableUponPush())) {
```

Comment:
nit: unnecessary parens

Reply:
My goodness. I did this PR in such a rush as I was heading out of the office. That's embarrassing 😂